development.ini

```diff
@@ -5,7 +5,10 @@ unicore.content_repos =
     unicore_frontend_straighttalk_za
     unicore_frontend_ureport_za
     unicore_frontend_mariestopes_za
-    unicore_frontend_ffl_za
+
+iogt.content_section_url_overrides =
+    ffl = http://za.ffl.qa-hub.unicore.io/
```

Comment:
From my discussion with Jonathan, the call-to-action blocks are external links to the individual websites.